### ISSUE TITLE
FR: [V2 Recommendations] - Video in Thumbnails

### ISSUE THREAD

**Issue by ZoeBeykirch** [hlxsites/vg-macktrucks-com#753]
### Context/Description

As a Mack Trucks content author, I want to add video content into the V2 Cards block so that I can enrich the recommended bulldog magazine articles and other web pages with more engaging multimedia content that will enhance the user experiences. 

### Acceptance Criteria

This ticket should be picked up after https://github.com/hlxsites/vg-macktrucks-com/issues/750

**User AC:**
- [ ] As a website user, I expect to see thumbnail video clips in the recommended article section while I'm reading a Bulldog Magazine article. 

**Editor AC:**
- [ ] As a content editor, I can add this block to any page. 
- [ ] As a content editor, I can use this block from the AEM Sidekick Library. 

**Non-functional AC:**
- [ ] The block variant will be named: "V2 Recommendations"
- [ ] The block can use a 30 second video clip without sound. 
- [ ] The block will use custom controls: play/pause
- [ ] The block will be responsive so that they display is correct across different devices and screen sizes. 
- [ ] The videos should load efficiently without significant buffering or delays. 
- [ ] The block can be used in #603
- [ ] The video will come from the metadata within the article

### Out of scope

- AEM Asset integration is not required if it will cause a blocker. 
- Videos with sound, therefore no mute/unmute functionality is required. 
- Autoplay (can be handled in a separate ticket if needed).

### Attachment / link to Figma

Desktop: https://www.figma.com/design/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?m=dev&node-id=14119-2407

![Image](https://github.com/hlxsites/vg-macktrucks-com/assets/125895595/33399110-958d-402c-a546-11b924d50020)

Mobile: https://www.figma.com/design/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?m=dev&node-id=2222-52924

![Image](https://github.com/hlxsites/vg-macktrucks-com/assets/125895595/9e15226a-8bb1-4b30-9f58-e96e23971c2c)

**Comment by ZoeBeykirch** [comment]
This ticket can be picked up after https://github.com/hlxsites/vg-macktrucks-com/issues/750 is delivered

**Comment by BeckyMedlin** [comment]
Need Adobe's support on this ticket @ZoeBeykirch @mollyaljenkins @Rapostu @verareljic It is still blocked due to video functionality issues.

**Comment by amglebihan** [comment]
This ticket is currently blocked, pending Adobe’s completion of the video-related work for Volvo Trucks. Once that is done, we can integrate the code into Mack Trucks, enabling the required features to move forward with this ticket.
